### ISSUE TITLE
@stylexjs/eslint-plugin: Confusing error message when a value is disallowed with `null`

### ISSUE THREAD

**Issue by tjosepo** [facebook/stylex#282]
**The problem**

When disallowing a property with the ESLint plugin, the error message is:

```
{property} value must be one of:
{reason}
```

While not completely incorrect, it's slightly misleading to say `value must be one of:` when there is no valid value.

**How to reproduce**

**ESLint configuration**

![image](https://github.com/facebook/stylex/assets/44372776/658d13cd-d797-4bd1-9576-2355644b7e93)

**Error message**

![image](https://github.com/facebook/stylex/assets/44372776/d422ba03-a5f5-49fe-867f-3c9678d3176b)

**Expected behavior**

I expect the error message to clearly indicate that the property is disallowed.

**Environment (include versions). Did this work in previous versions?**
```
@stylexjs/eslint-plugin@0.4.1
```

**Comment by nmn** [comment]
@tjosepo Yes, this is a known issue. You're welcome to take a crack at it, but I wouldn't say it's the easiest issue to tackle!
